integration-cli/docker_cli_save_load_unix_test.go

```diff
@@ -71,6 +72,9 @@ func (s *DockerCLISaveLoadSuite) TestSaveAndLoadRepoStdout(c *testing.T) {
 }
 
 func (s *DockerCLISaveLoadSuite) TestSaveAndLoadWithProgressBar(c *testing.T) {
+	// TODO(vvoland): https://github.com/moby/moby/issues/43910
+	skip.If(c, testEnv.UsingSnapshotter(), "TODO: Not implemented yet")
```

Comment:
Added a skip, because it's just a visual nuisance which doesn't affect the save/load itself and we have a tracking issue for  it.

Let me know if you think we should leave this one failing until we really fix it.

Reply:
Let's leave it failing for now, we should only skip those that don't make sense with containerd, and this one looks like a low hanging fruit?

Reply:
It's not that easy, as we need to show the progress for each imported layer (like pull) which isn't directly exposed by the containerd's `archive.ImportIndex`.

Reply:
For reference, this is the progress that's missing:
```
$ docker load -i b.tar
3694737149b1: Loading layer [==================================================>]  1.917MB/1.917MB
Loaded image ID: sha256:1e9bbbbb28889a02334e8003b19c81bd7fc778d9257a82cd308730890fe2c4a7
```

Currently with c8d it's only:
```
$ docker load -i b.tar
Loaded image ID: sha256:1e9bbbbb28889a02334e8003b19c81bd7fc778d9257a82cd308730890fe2c4a7
```

Reply:
OK let's not skip this one for now, we might be able to use the same thing we use to track pull progress.